packages/core/src/types/IViewport.ts

```diff
@@ -110,17 +110,76 @@ export type ReferencedImageRange = ViewReference & {
+ * The most degenerate case has a single point, and includes all planes
+ * which pass through that point.  For this case, if a view plane normal
+ * is required to display the point, the current view plane normal of the
+ * viewport is recommended, or the acquisition plane if there is no view plane
+ * normal.
+ *
+ * A colinear plane restricts the set of all planes containing point to those
+ * which are around the line from point to point+inPlaneVector1
+ * If a view plane normal is required to display the line, the current view up
+ * cross product with the inPlaneVector1 is recommended if these are not colinear.
+ * Otherwise, the cross product of inPlaneVector1 and the current view plane normal
+ * is recommended.
+ *
+ * A specific plane reference further restricts the colinear planes to those
+ * which also include point+inPlaneVector2
+ * The view plane normal to display this point is required to be the cross product
+ * of the two inPlaneVector values.
+ */
+export type ReferencedPlane = {
+  FrameOfReferenceUID: string;
+
+  /** A single point within the reference plane is required for all references */
+  point: Point3;
+
+  /** An inPlaneVector1 is required for all colinear referenced planes.
+   * Shall not be undefined if inPlaneVector2 is defined.
+   */
+  inPlaneVector1?: Point3;
+
+  /** An inPlaneVector2 is required for all full planar definitions.
+   * Shall have a non-zero dot product with inPlaneVector1, that is, shall be
+   * non-colinear with inPlaneVector1.
+   */
+  inPlaneVector2?: Point3;
+};
+
 /**
  * A view reference references the image/location of an image.  Typical use
  * cases include remembering the current position of a viewport to allow returning
  * to it later, as well as determining whether specific views should show annotations
  * or other overlay information.
+ *
+ * Note this is an interface as it is designed to allow extension/customization
+ * by additional Viewport modules.
  */
-export type ViewReference = {
+export interface ViewReference {
   /**
    * The FrameOfReferenceUID
    */
   FrameOfReferenceUID?: string;
+
+  /**
+   * A referenced plane identifies one or more planes.
+   * Currently this has a point within the plane to identify the focal depth
+   * (but NOT the focal point), and up to two coplanar vectors.
+   *
+   * The referenced plane is visible depth wise if the point minus the focal point of hte
+   * current view is orthogonal to the view plane normal of the viewport.
+   * This is sufficient for a single-point identifier.
+   * For a line, the inPlaneVector must be orthogonal to the view plane normal of the viewport.
+   * For a planar annotation, both inPlaneVectors must be orthogonal to the view plane normal of the viewport.
+   *
+   * This extension from the least specific to the most specific types of views allows
+   * determining whether a view can be seen in a variety of conditions.  It does not
+   * allow recovering the original view reference.
+   */
```

Comment:
is it one plane or more than one plane? the ts type says otherwise

Reply:
Resolved this by renaming and changing the docs.